### PR TITLE
add syslog-adapter

### PR DESCRIPTION
**Pull request by LinuxGit**

### CODE REVIEW

File: alerts/syslog-adapter/syslog.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		case "firing":
+			syslogMsg.Status = "0"
+		case "resolved":
+			syslogMsg.Status = "1"
+		default:
+			syslogMsg.Status = "1"
+		}
+
+		switch syslogMsg.AlertName {
+		case "TiDB_is_Down", "TiKV_is_Down", "PD_is_Down":
```
Comment (jomenxiao) [inline alerts/syslog-adapter/syslog.go]: why "*_is_Down"? `AlertName `  is not hardcode.

File: alerts/syslog-adapter/syslog.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+			Status:      at.Status,
+			Time:        at.StartsAt.Format(timeFormat),
+		}
+
+		syslogMsg.Level = *alertLevel
+		ip := *alertIP
+		hostname := *alertHostname
+
+		switch syslogMsg.Status {
+		case "firing":
```
Comment (jomenxiao) [inline alerts/syslog-adapter/syslog.go]: 0 mean?

Comment (LinuxGit) [reply alerts/syslog-adapter/syslog.go]: customer's  requirements: firing: 0, resolved: 1.

**Comment by jomenxiao** [comment]
LGTM

**Pull request merged**